runner/ansible/roles/1.4.1.monitor_interval/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.4.1.monitor_interval'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    if cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1; then
```

Comment:
Cannot we do something to have all these `1.4.1` tests as an unique test?
This will make things really hard to work with, because they would be treated as individual tests, something we don't really like to do.
I would try to run all the commands here and compare them with the expected results or something.
It should be doable with shell,
Maybe, store the return code of the comparisons in an array, and check eventually if there is some incorrect value.